test/unit/ui/control/scale.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const test = require('mapbox-gl-js-test').test;
+const window = require('../../../../src/util/window');
+const Map = require('../../../../src/ui/map');
+const config = require('../../../../src/util/config');
+const ScaleControl = require('../../../../src/ui/control/scale_control');
+
+function createMap() {
+    const container = window.document.createElement('div');
+    config.ACCESS_TOKEN = 'pk.123';
```

Comment:
The dummy access token shouldn't be necessary, and nor should the `owner` or `id` properties below.  (I think it exists in the `AttributionControl` tests because that specific control needs it.)